napari/util/misc.py

```diff
@@ -58,6 +59,135 @@ def is_rgb(shape):
         return False
 
 
+def is_pyramid(data):
+    """If data is a list of arrays of decreaing size.
```

Comment:
decreasing. Is this a butterfly keyboard thing? =P

Reply:
:-) fixed - i copy and paste a lot??